test/screenshot/text-field/TextFieldPermutations.tsx

```diff
@@ -13,7 +13,11 @@ import {
 import TestField from './TestTextField';
 
 const textFields = (variantProps: {variant?: string}) => {
-  return iconsMap.map((icon: {leadingIcon?: React.ReactNode, trailingIcon?: React.ReactNode}) => {
+  return iconsMap.map((icon: {
```

Comment:
also getting an error when click on the leading-trailing-icon permutation. Try clicking on one of the icons here:
![tndrewahb7u](https://user-images.githubusercontent.com/579873/50852974-e6547580-1335-11e9-913a-c8a279f17950.png)

They also should not have `tabIndex` at all. It should be undefined.

Reply:
cool - i think one of your other commits must have fixed it